packages/manager/apps/dedicated/client/app/css/source.less

```diff
@@ -20,7 +17,7 @@
 @import "less/common/buttonIcons";
 @import "less/common/button";
 @import "less/common/countryIcon";
-@import "less/common/datetimepicker";
+// @import "less/common/datetimepicker";
```

Comment:
Could be removed then?

Reply:
I hope, I'm searching if it is used or not. If no, we could remove `bootstrap-datetimepicker` too https://github.com/ovh-ux/manager/blob/master/packages/manager/apps/dedicated/package.json#L87

Reply:
Not used, removed.